fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -257,6 +257,171 @@ sebasedBiasFieldMNI="$SubjectFolder/$AtlasSpaceFolder/Results/$NameOffMRI/${Name
 
 fMRIFolder="$Path"/"$Subject"/"$NameOffMRI"
 
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+
+PreregisterTool=`opts_GetOpt1 "--preregistertool" $@`                    # what to use to preregister BOLDs before FSL BBR - epi_reg (default) or flirt
+DoSliceTimeCorrection=`opts_GetOpt1 "--doslicetime" $@`                  # Whether to do slicetime correction (TRUE), FALSE to omit
+SliceTimerCorrectionParameters=$(opts_GetOpt1 "--slicetimerparams" "$@") # A '@' separated list of FSL slicetimer options. Please see FSL slicetimer documentation for details.
+                                                                         # Verbose (-v) is already turned on. TR is read from 'pixdim4' of the input NIFTI itself.
+                                                                         # e.g. --slicetimerparams="--odd@--ocustom=<CustomInterleaveFile>"
+BOLDMask=`opts_GetOpt1 "--boldmask" $@`                                  # Specifies what mask to use for the final bold:
+                                                                         #   T1_fMRI_FOV: combined T1w brain mask and fMRI FOV masks (the default), 
+                                                                         #   T1_DILATED_fMRI_FOV: a once dilated T1w brain based mask combined with fMRI FOV
+                                                                         #   T1_DILATED2x_fMRI_FOV: a twice dilated T1w brain based mask combined with fMRI FOV, 
+                                                                         #   fMRI_FOV: a fMRI FOV mask
+                                                                         #   NOTE: When using SEBASED bias field correction, the mask will default to T1_fMRI_FOV
```

Comment:
This comment is unclear to me.  `T1_fMRI_FOV` is already the default setting.  Are you saying that when using SEBASED bias field correction that the `--boldmask` option is _irrelevant_ and `T1_fMRI_FOV` mode will always be used, regardless of the `--boldmask` setting?

Reply:
Yes, that's what I'm trying to say. SEBASED bias field is T1 masked so applying it effectively makes any other mask that is larger than T1 mask irrelevant.

Reply:
Then we should say that explicitly.  Also, if that is the case, isn't the whole mask option somewhat irrelevant, since SEBASED is the bias field correction mode that people should be using?

Reply:
In legacy data SE images are a rarity so for most of them there are no SEBASED bias fields. Also I'd rather make SEBASED bias field nonmasked—if possible—than remove the mask options. I have not had the time and am not familiar with the code enough to enable that. @coalsont, would that be possible?

Reply:
@mharms How would you state that more explicitly?

Reply:
Not sure, @glasserm is the one who came up with the idea of the SEBASED bias field.  It basically compares the gradient echo and spin echo images, as I recall.

Reply:
The last couple of lines of the script produce mildly dilated bias files.  One could dilate to fill the files with fslmaths -dilall.

Reply:
I'm confused here.  `BOLDMask` only seems to get used in `IntensityNormalization`.  So what exactly is the relationship of that to the SEBASED bias field correction?

Reply:
@glasserm would you be ok with me changing the two last lines 87, 88 of `ComputeSpinEchoBiasField.sh` to `-dilall`?

Reply:
@mharms in line 189 in Intensity normalization:
```
${FSLDIR}/bin/fslmaths ${InputfMRI} $biascom $jacobiancom -mas ${FinalMask} -thr 0 -ing 10000 ${OutputfMRI} -odt float
```

`$biascom` expands to `biascom="-div $BiasField"` (line 121). If the BiasFiled is masked, then the mask that was used for the `$BiasField` is propagated to the BOLD image as anything that is outside of the masked `$BiasField` will be set to 0.

Reply:
That would not really have the expected effect as that small dilation is just for a trilinear resampling that is immediately masked again.  One would need to change how that remasking is done here: in fMRIVolume.sh lines 426-452

Reply:
@glasserm Then it might be simplest to just do it at the very end in `IntensityNormalisation.sh` within the `if` check (line 121). In this case the biasfield would be dilated just before being used.

Reply:
Yes you could do some dilation there if needed.

Reply:
Ok. I'll add:

```
biascom=""
if [[ "$BiasField" != "" ]]
then
    ${FSLDIR}/bin/fslmaths ${BiasField} -dilall ${BiasField}_dilated
    biascom="-div ${BiasField}_dilated"
fi
```

Reply:
And I removed the warning note about mask use, which is not necessary anymore.

Reply:
Do we want to keep this `${BiasField}_dilated` file around once we are done with it?

Reply:
Well, it is the record of what was used. But I'll refer the decision to your team. I don't have a clear idea or guidance on what needs to be retained and what not.

Reply:
It would be fine to keep it.

Reply:
If that's the case, then the PR is ready for pulling in.